src/sql/parts/insights/browser/insightsDialogView.ts

```diff
@@ -211,8 +213,13 @@ export class InsightsDialogView extends Modal {
 			});
 		}));
 
-		this._splitView.addView(topTableView);
-		this._splitView.addView(bottomTableView);
+		attachPanelStyler(topTableView, this._themeService);
+		attachPanelStyler(bottomTableView, this._themeService);
+
+		panelCreation.then(() => {
+			this.panelView.addPanel(topTableView, 1, 0);
```

Comment:
>1 [](start = 41, length = 1)

Why size is 1 here?

Reply:
I think this size is a relative size value, not a pixel size